extended/pom.xml

```
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>client-java-extended</artifactId>
+    <packaging>bundle</packaging>
+    <name>client-java-extended</name>
+    <version>5.0.0-SNAPSHOT</version>
+    <url>https://github.com/kubernetes-client/java</url>
+
+    <parent>
+        <artifactId>client-java-parent</artifactId>
+        <groupId>io.kubernetes</groupId>
+        <version>5.0.0-SNAPSHOT</version>
+    </parent>
+
```

```suggestion
<dependencies>
        <dependency>
            <groupId>io.kubernetes</groupId>
            <artifactId>client-java-api</artifactId>
            <version>${project.version}</version>
        </dependency>
        <dependency>
            <groupId>io.kubernetes</groupId>
            <artifactId>client-java-proto</artifactId>
            <version>${project.version}</version>
        </dependency>
</dependencies>
```

better to add some common dependencies here beforehand to avoid conflict?

---

yep, makes sense to me.

---

done~ ptal